CHANGELOG.md

```diff
@@ -1,6 +1,13 @@
+# Staged to Develop
+## Fixes
+- Addresses potential import shenanigans for some operating systems
```

Comment:
> shenanigans

Probably shouldn't use such colloquial language in official documentation. Would recommend changing to "issues" or "problems"

Reply:
Yeah, that makes sense - I'll patch that real quick